src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -25,6 +25,14 @@ namespace Microsoft.VisualStudio.Packaging
         new string[] { "dotnetcore" },
         new string[] { "SolutionHasProjectCapability:.NET & CPS" }
         )]
+    // Tell the diff window that these files should use the XML editor
+    [ProvideLanguageExtension("{f6819a78-a205-47b5-be1c-675b3c7f0b8e}", ".csproj")]
+    [ProvideLanguageExtension("{f6819a78-a205-47b5-be1c-675b3c7f0b8e}", ".vbproj")]
+    [ProvideLanguageExtension("{f6819a78-a205-47b5-be1c-675b3c7f0b8e}", ".fsproj")]
+    // Tell the diff window that these files are supported and should be editable. The actual content type doesn't matter for this
+    [ProvideDiffSupportedContentType(".csproj", "")]
+    [ProvideDiffSupportedContentType(".vbproj", "")]
+    [ProvideDiffSupportedContentType(".fsproj", "")]
```

Comment:
What does empty mean here?

Reply:
I see the comment now, just rename the attribute to make this clear and then you don't need a comment.

Reply:
Perhaps I'm misunderstanding what you mean, but I don't know what you would rename the attribute to, it follows the pattern of the other Provide* attributes. If I remove the comment, then 6 months from now someone will be wondering what an empty string means and will have to go digging through code. There are many times where I wish there would have been more explanatory comments in our repo.

Reply:
The comment exists because the attribute name isn't descriptive enough`. ProvideDiffSupportedExtension` with no content type would remove the need for the comment and wouldn't have me questioning why we're passing empty string here.